src/components/GrowlNotification/index.js

```diff
@@ -36,6 +21,23 @@ function GrowlNotification(_, ref) {
     const [bodyText, setBodyText] = useState('');
     const [type, setType] = useState('success');
     const [duration, setDuration] = useState();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    const types = {
```

Comment:
should we memoize this one?

Reply:
I believe we should

Reply:
@lukemorawski could you address this comment?

Reply:
@fabioh8010 I still don't understand why I should memoize this :)